server/forge/forge.go

```diff
@@ -28,6 +29,12 @@ import (
 
 // TODO: use pagination
 
+type Service interface {
```

Comment:
had to adjust it a bit to prevent cyclic imports, but now there are functions in the service manager to get a forge from a repo, user or the main one. Is this how you would expect it?